app/views/courses/_advice.html.erb

```diff
@@ -1,10 +1,21 @@
 <div class="govuk-!-margin-bottom-8">
   <h3 class="govuk-heading-l" id="section-advice">Support and advice</h3>
-  <p class="govuk-body">For questions about this course you should contact the training provider using <%= govuk_link_to 'the contact details above', '#contact_section' %>.</p>
 
-  <h4 class="govuk-heading-m">Get support and advice about teaching</h4>
-  <p class="govuk-body">You can speak to a <%= govuk_link_to t('service_name.get_into_teaching'), t('get_into_teaching.url_get_an_advisor') %> adviser for free. They’re all experienced teachers who can help you prepare your application, book school experience, and access exclusive teaching events.</p>
-  <p class="govuk-body">You can also call <%= t('service_name.get_into_teaching') %> free of charge on <%= t('get_into_teaching.tel') %>, or speak to an adviser using their <%= govuk_link_to 'online chat service', t('get_into_teaching.url_online_chat') %> <%= t('get_into_teaching.opening_times') %>.</p>
+<!--  <p class="govuk-body">Register with <%#= govuk_link_to t('service_name.get_into_teaching'), t('get_into_teaching.url_get_an_advisor') %>,-->
```

Comment:
think this needs to be uncommented.

Reply:
Just spotted that, it needed to come out. I'd left it in to steal all the i18n strings. It's gone now

Reply:
sorry outdated now!